lib/backend/k8s/conversion/workload_endpoint_default.go

```diff
@@ -64,6 +64,19 @@ func (wc defaultWorkloadEndpointConverter) PodToWorkloadEndpoints(pod *kapiv1.Po
 	return []*model.KVPair{wep}, nil
 }
 
+func SanitizeServiceAccountName(n string) string {
+	if len(n) > 63 {
+		// If the serviceaccount name is longer than the length allowed for a label,
+		// we need to trim it down so that it meets validation, but add a hash of the original
+		// value to maintain uniqueness in selectors.
+		prefix := strings.TrimRight(n[0:50], "-")
```

Comment:
what if the dash was like the second char.  i know it works, but it's not a nice abbreviation.